Examples/Tests/effective_potential_electrostatic/analysis.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import dill
+import matplotlib.pyplot as plt
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+from scipy.interpolate import RegularGridInterpolator
```

Comment:
Could you add a high-level text description that summarizes what this Python checks (e.g. density after the expansion of the plasma)?